free-programming-books.md

```diff
@@ -935,8 +935,13 @@ Kerridge (PDF) (email address *requested*, not required)
 ### Arduino
 
 * [Arduino Programming Notebook](http://www.lulu.com/shop/brian-evans/arduino-programming-notebook/ebook/product-18598708.html) - Brian Evans
+* [Arduino Tips, Tricks, and Techniques](https://cdn-learn.adafruit.com/downloads/pdf/arduino-tips-tricks-and-techniques.pdf) - lady ada (PDF)
+* [Getting Started with Arduino](http://phylab.fudan.edu.cn/lib/exe/fetch.php?media=yuandi:arduino:getting_started_with_arduino_v2.pdf) - Massimo Banzi (PDF)
```

Comment:
Not an authorized distribution. Please see CONTRIBUTING.md guidelines.

Reply:
Removed this one book.